libraries/mediaupload/impl/src/main/kotlin/io/element/android/libraries/mediaupload/impl/ImageCompressor.kt

```diff
@@ -36,7 +36,7 @@ class ImageCompressor @Inject constructor(
         resizeMode: ResizeMode,
         format: Bitmap.CompressFormat = Bitmap.CompressFormat.JPEG,
         orientation: Int = ExifInterface.ORIENTATION_UNDEFINED,
-        desiredQuality: Int = 80,
+        desiredQuality: Int = 78,
```

Comment:
This is an oddly specific number 😅 . I'm curious about how we decided on 78 instead of 75 or 80.

Reply:
Oh, I see, it's the same as in other clients. Solved!